corehq/apps/receiverwrapper/tests/test_submissions.py

```diff
@@ -82,6 +82,10 @@ def _test(self, form, xmlns):
         expected = self._get_expected_json(xform_id, xmlns)
         self.assertEqual(foo, expected)
 
+    def _test_case_supports_non_bmp_chars(self, case_id):
```

Comment:
I'm curious why this needs to be its own function instead of part of `test_submit_with_non_bmp_chars`

Reply:
I was also confused about that, I chose to go with creating another function because I thought rest of the tests were a pattern where testing logic is wrapped inside a function.
Now after you mentioned I am also thinking about why something that will be exactly used once should be in a function 😅 
I will move it inside `test_submit_with_non_bmp_chars`.